docs/examples/viz_brownian_motion.py

```diff
@@ -0,0 +1,117 @@
+"""
+======================================================================
+Brownian motion
+======================================================================
+Brownian motion, or pedesis, is the random motion of particles suspended in
+a medium. In this animation, path followed by 4 particles exhibiting brownian
+motion in 3D is plotted.
+
+Importing necessary modules
+"""
+
+from fury import window, actor, ui
+import numpy as np
+from scipy.stats import norm
+
+np.random.seed(8)
+
+
+###############################################################################
+# Variable(s) and their description-
+# total_time: time to be discretized via time_steps (default: 5)
+# num_total_steps: total number of steps each particle will take (default: 300)
+# time_step = (total_time/num_total_steps): time step (default: 5/300)
+# counter_step: to keep track of number of steps taken (initialised to 0)
+# delta: delta determines the "speed" of the Brownian motion. Increase delta
+#        to speed up the motion of the particle(s). The random variable
+#        of the position has a normal distribution whose mean is the position
+#        at counter_step=0 and whose variance is delta**2*time_step.
+#        (default: 1.8)
+# num_particles: number of particles whose path will be plotted
+# common_origin: common origination point of the particles
+# coords: list of origination coordinates of the particles
+# colors: list of colors of the particles
+
+total_time = 5
+num_total_steps = 300
+time_step = total_time/num_total_steps
+counter_step = 0
+delta = 1.8
+num_particles = 4
+common_origin = np.array([0, 0, 0])
+coords = [common_origin for i in range(num_particles)]
+colors = [window.colors.red, window.colors.blue, window.colors.yellow,
+          window.colors.green]
+
+###############################################################################
+# Function that generates the path of the particles
+
+
+def generatepath(point, delta, time_step):
+    x, y, z = (point[i] for i in range(3))
+    x += norm.rvs(scale=delta**2*time_step)
+    y += norm.rvs(scale=delta**2*time_step)
+    z += norm.rvs(scale=delta**2*time_step)
+    return x, y, z
+
+
+###############################################################################
+# Creating a scene object and configuring the camera's position
+
+scene = window.Scene()
+scene.background((1.0, 1.0, 1.0))
+scene.zoom(2.1)
+scene.set_camera(position=(0, 0, 40), focal_point=(0.0, 0.0, 0.0),
+                 view_up=(0.0, 0.0, 0.0))
+showm = window.ShowManager(scene,
+                           size=(600, 600), reset_camera=True,
+                           order_transparent=True)
+showm.initialize()
+
+
+###############################################################################
+# Creating a container (cube actor) inside which the particle(s) move around
+
+center1 = np.array([[0, 0, 0]])
```

Comment:
this variable is unnecessary, you can feed the point directly on your cube actor if you have only 1 center